cmd/kubeadm/app/preflight/checks_linux.go

```diff
@@ -42,3 +45,13 @@ func (idsc IsDockerSystemdCheck) Check() (warnings, errorList []error) {
 	}
 	return warnings, nil
 }
+
+// Check determines if IPVS proxier can be used or not
+func (ipvspc IPVSProxierCheck) Check() (warnings, errors []error) {
```

Comment:
It doesn't look like we return warnings in any way, can we remove the return argument?

Reply:
I don't think we can as this is a part of [Checker interface](https://github.com/kubernetes/kubernetes/blob/18ac575d9be27dbfbd464557b85615bbabfa69ba/cmd/kubeadm/app/preflight/checks.go#L88:L91). If we do that it would break kubeamd build I believe.